src/app/custom/translations.tsv

```diff
@@ -1091,3 +1091,10 @@
 "binary"	"Binary"	"Binary"
 "slicedice"	"Slice and Dice"	"Slice and Dice"
 "treemap_ratio"	"Aspect ratio of the layout method"	"Rapport d'aspect de la méthode de présentation"
+"formatEJS"	"Text - JavaScript/HTML templating"	"Texte - JavaScript/HTML templating (modèle)"
+"formatEJSDescription"	"Create template with EJS (HTML + JavaScript) and add you data in with the LODEX routines"	"Créez un modèle avec EJS (HTML + JavaScript) et ajoutez vos données avec les routines LODEX."
+"ejs_documentation"	"EJS Documentation"	"Documentation EJS"
+"lodash_documentation"	"Lodash Documentation"	"Documentation Lodash"
+"ejs_variable_list"	"You can use these variables to access to the data and utils:"	"Vous pouvez utiliser ces variables pour accédez au données et au utilitaire :"
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
"ejs_variable_list"	"You can use these variables to access to the data and utils:"	"Vous pouvez utiliser ces variables pour accéder aux données et aux utilitaires :"
```